cmd/thv/app/run.go

```diff
@@ -124,6 +125,7 @@ func init() {
 		"",
 		"Path to a custom CA certificate file to use for container builds",
 	)
+	runCmd.Flags().BoolVarP(&runSecure, "secure", "s", false, "Run in secure mode (blocks if container fails verification)")
```

Comment:
should we change the behavior here?

let's have a flag called `image-verification` that can have three values:
* `enabled`: fails the deployment if it doesn't pass.
* `disabled`: always deploys the MCP without doing this check.
* `warn`: Does the verification, warns you if it fails, but continues anyway.

The default would be `warn`.

what do you think?

Reply:
I like it 👍